plugins/tangle/solidifier.go

```diff
@@ -211,13 +222,15 @@ func solidQueueCheck(milestoneIndex milestone_index.MilestoneIndex, milestoneTai
 
 				if newlySolid && tangle.IsNodeSynced() {
 					// Propagate solidity to the future cone (txs attached to the txs of this milestone)
+					entryTx.RegisterConsumer() //+1
 					gossipSolidifierWorkerPool.Submit(entryTx)
```

Comment:
Where is the Release? Is it in the Worker?

Reply:
yes, workers should be submitted with a registered consumer